packages/vite/types/importMeta.d.ts

```diff
@@ -44,19 +44,8 @@ interface ImportMeta {
 
   readonly env: ImportMetaEnv
 
-  glob(pattern: string): Record<
-    string,
-    () => Promise<{
-      [key: string]: any
-    }>
-  >
-
-  globEager(pattern: string): Record<
-    string,
-    {
-      [key: string]: any
-    }
-  >
+  glob<T = {[key: string]: any}>(pattern: string): Record<string, () => Promise<T>>
```

Comment:
I think the formatting will be with spaces. Could you run prettier for this file?
After that, count me as approved

Reply:
I have ran prettier now.

Once more thing should we name the generic type something other than `T`?
Perhaps `Module` or `ModuleType`?

Reply:
Sounds good 🙂  
I will let you decide what is better, I like both

Reply:
Done :)